__tests__/app.test.js

```diff
@@ -31,5 +31,39 @@ describe('/api/categories', () => {
                 })
             })
         })
+        describe('GET /api/reviews/:review_id', () => {
+            test('happy path', () => {
+                return request(app).get('/api/reviews/1').expect(200).then(({body}) => {
+                    expect(Array.isArray(body)).toBe(false)
+                    expect(typeof body).toBe('object')
+
+                    expect(Object.keys(body).length).toBe(9)
+
+                    expect(body).toEqual(
+                        expect.objectContaining({
+                            review_id: expect.any(Number),
+                            title: expect.any(String),
+                            review_body: expect.any(String),
+                            designer: expect.any(String),
+                            review_img_url: expect.any(String),
+                            votes: expect.any(Number),
+                            category: expect.any(String),
+                            owner: expect.any(String),
+                            created_at: expect.any(String)
+                        })
```

Comment:
Instead of doing any number/string/etc here, as you are requesting a specific article, i would like you to make sure you are getting the exact correct data back and not just the right shape by checking it against the data that you are feeding into the database when seeded. You can find this info in the data directory of your project

Reply:
Ok will do! I did have that originally but wasn't sure if we might end up changing the default sort order down the line - Will amend to specific data now thanks